Source/PeleLMeX_Advance.cpp

```diff
@@ -74,6 +74,9 @@ PeleLM::Advance(int is_initIter)
 
   for (int lev = 0; lev <= finest_level; lev++) {
     m_extSource[lev]->setVal(0.);
+    if (m_n_sparks > 0) {
+      addSpark(lev, AmrNewTime);
```

Comment:
Is this needed? We don't have an equivalent source term call for soot/spray/rad.

Reply:
Doesn't look like it - removed.